.gitignore

```diff
@@ -52,3 +52,6 @@ Thumbs.db
 
 # eslint
 .eslintcache
+
+# test proxy
+dev.proxy.conf.json
```

Comment:
Why this new file ? Not necessary ?

Reply:
For testing the ui with the proxy, it is not absolutely necessary but this way we don't need to recreate a file everytime we need to quick-test something